server/src/tenant/entity-resolver/utils/generate-args-input.util.ts

```diff
@@ -0,0 +1,28 @@
+import { stringifyWithoutKeyQuote } from './stringify-without-key-quote.util';
+
+export const generateArgsInput = (args: any) => {
+  let argsString = '';
+
+  for (const key in args) {
+    // Check if the value is not undefined
+    if (args[key] !== undefined) {
```

Comment:
Small suggestion: invert "if" statement here and when possible to avoid nesting. I think it makes the code easier to read in general. This and the "early return pattern". Up to you!